my_gazebo_concert/src/solutions/concert.launch

```diff
@@ -0,0 +1,17 @@
+<launch>
```

Comment:
Since you won't be overlaying the original anymore you can use the original launch file with the argument from your tutorial package `roslaunch gazebo_concert concert.launch gazebo_services:=my_gazebo_concert/gazebo.services`